src/rgw/rgw_rest_s3.cc

```diff
@@ -731,12 +731,10 @@ class RGWSetBucketVersioningParser : public RGWXMLParser
 
 int RGWSetBucketVersioning_ObjStore_S3::get_params()
 {
-#define GET_BUCKET_VERSIONING_BUF_MAX (128 * 1024)
-
   char *data = nullptr;
   int len = 0;
   int r =
-    rgw_rest_read_all_input(s, &data, &len, GET_BUCKET_VERSIONING_BUF_MAX);
+    rgw_rest_read_all_input(s, &data, &len, s->cct->_conf->rgw_max_put_param_size, false);
```

Comment:
this was github's folding that was the culprit, we further reuse the var. in the function body